tests/conftest.py

```diff
@@ -13,6 +13,41 @@
 from jupyterhub_traefik_proxy import TraefikTomlProxy
 
 
+@pytest.fixture
+async def autohttps_toml_proxy():
+    """Fixture returning a configured Let's Encrypt TraefikTomlProxy"""
+    proxy = TraefikTomlProxy(
+        public_url="http://127.0.0.1:8000",
```

Comment:
I'm not sure about what to test, but perhaps this should be `https://jupyter.test:8443`?

Reply:
I think you're right (checkout [this comment](https://github.com/jupyterhub/traefik-proxy/pull/108#discussion_r427219235) for what was going on in my head about `public_url` :sweat_smile:)